proto/zetachain/zetacore/observer/operational.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package zetachain.zetacore.observer;
+
+option go_package = "github.com/zeta-chain/node/x/observer/types";
+
+message OperationalFlags {
+  // Height for a coordinated zetaclient restart.
+  // Will be ignored if missed.
+  int64 restart_height = 1;
```

Comment:
Not a strong opinion. Are `zetaclient_restart_height` and `OperationParams` also good names? It's up to you.

Reply:
It's already under the `observer` section so doesn't that imply that it's zetaclient functionality.